libfwupd/fwupd-remote.c

```diff
@@ -416,6 +421,12 @@ fwupd_remote_load_from_filename (FwupdRemote *self,
 		return FALSE;
 	}
 
+	/* override remote kind if explict */
+	remote_kind = g_key_file_get_string (kf, group, "Type", NULL);
+	if (remote_kind != NULL) {
+		priv->kind = fwupd_remote_kind_from_string (remote_kind);
+	}
+
```

Comment:
I think specifically keeping this block in, and putting the type into remotes in tree that are missing it makes sense (drop the override remote kind comment though)